packages/block-editor/src/components/global-styles/style-system/manager.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import createEmotion from 'create-emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+export const globalStylesManager = createEmotion( {
+	key: 'wp-gs',
+	speedy: true,
+} );
+
+export const getGlobalStylesSheetAsCssStringSync = () => {
```

Comment:
I think this and the following functions should use standard named function declarations rather than anonymous functions, since named functions show their names in stack traces, making debugging easier. There's no reason for these to be anonymous functions, as far as I can tell. In fact, I think a standard function declaration would use fewer characters in this instance.